plugins/permanentpen.md

```diff
@@ -0,0 +1,128 @@
+title_nav: Permanent Pen
+description: Apply formats while typing
+keywords: permanent pen copy text format style
+controls: toolbar button, contextmenu, menu item
+---
+
+The Permanent Pen allows a user to apply a pre-defined format (color, typeface and type style, or font) to a desired section of content overriding the default values set for that content area.
+
+The Permanent Pen is useful when a distinctly styled text needs to be repeatedly added to the content. The user can pre-define the desired style in the Permanent Pen properties. This pre-defined style is applied to all the text entered into the document with the Permanent Pen enabled. The ability to store information for reuse saves the user time and ensures consistency across the content.
+
+The Permanent Pen can also be used to highlight user notes and other useful comments in a different format for easy identification. This allows the user to add comments without modifying the original content.
+
+The default properties of Permanent Pen are:
+
+```js
+fontname: 'arial,helvetica,sans-serif',
+forecolor: '#E74C3C',
+fontsize: '12pt',
+hilitecolor: '',
+bold: true,
+italic: false,
+underline: false,
+strikethrough: false
+```
+
+Permanent Pen is a premium plugin from Tiny. Please see the [Premium features]({{site.baseurl}}/enterprise/permanentpen/) section for all the buying options.
+
+Once you have obtained the Permanent Pen plugin, refer to the following instructions for using it.
+
+### Configuring the Permanent Pen toolbar button
+
+Use the following script to configure the Permanent Pen toolbar button:
+
+```js
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  plugins: 'permanentpen',
+  contextmenu: 'permanentpen',
+  toolbar: 'permanentpen'
+```
+**Result**:
+The ![**Permanent Pen**]({{site.baseurl}}/images/pp-disabled.png)(**Permanent Pen**) button appears in the toolbar menu.
+
+### Using a Permanent Pen
+
+#### To enable Permanent Pen
+
+1. Click on the ![**Permanent Pen**]({{site.baseurl}}/images/pp-disabled.png)(**Permanent Pen**) toolbar button to enable the Permanent Pen.
+2. Place the cursor in the desired text area and start typing.
+
+**Result**:
+The **Permanent Pen** toolbar button background color changes to _grey_ ![**Permanent Pen**]({{site.baseurl}}/images/pp-enabled.png) and the text is entered using the default settings.
+
+#### To disable Permanent Pen
+
+To exit the Permanent Pen, click on the ![**Permanent Pen**]({{site.baseurl}}/images/pp-enabled.png)(**Permanent Pen**) icon.
+
+The **Permanent Pen** toolbar button background color changes to _white_ ![**Permanent Pen**]({{site.baseurl}}/images/pp-disabled.png).
+
+### Accessing the Permanent Pen properties
+
+There are two ways to access the Permanent Pen properties:
+
+#### From the menubar
+
+In the TinyMCE 5.0 instance, choose the *Format* -> *(Permanent Pen)* option from the **Format** menu.
+
+![Format Menu]({{site.baseurl}}/images/formatmenu.png)
```

Comment:
Would be nice if we could get clean screenshots... Editor team or DX might be able to help if you want to do anything with that.